OJS.Workers.Common/Settings.cs

```diff
@@ -2,7 +2,7 @@
 {
     using System.Collections.Generic;
     using System.Linq;
-
+    using System.Runtime.InteropServices;
```

Comment:
Is this needed or just a left-over?

Reply:
Its needed for the RuntimeInformation.IsOSPlatform OS conditional